src/components/App.jsx

```diff
@@ -48,6 +48,7 @@ class App extends React.Component {
   componentDidMount() {
     this.props.dispatch(fetchProject());
     this.googleLogger.remember({ projectToken: 'antiSlaveryManuscripts' });
+    Rollbar.configure({ payload: { environment: env } });
```

Comment:
I'm wondering if we should also configure to record which user is reporting the error; this might help us with user comments on Talk.

But... that might be tricky since the User login check happens at a much later stage in the app life cycle. Can you run `Rollbar.configure()` _again_ at a later stage to add/edit values?

Reply:
Yes, you can add `Rollbar.configure()` at a later stage to adjust the params however you want.